src/main/java/org/opensearch/security/action/whoami/WhoAmIAction.java

```diff
@@ -31,7 +31,7 @@
 public class WhoAmIAction extends ActionType<WhoAmIResponse> {
 
     public static final WhoAmIAction INSTANCE = new WhoAmIAction();
-    public static final String NAME = "cluster:admin/opendistro_security/whoami";
+    public static final String NAME = "cluster:admin/opendistro_security/whoamiprotected";
```

Comment:
Is this change necessary?

Reply:
No.

Reply:
No. it's a protected version of `/whoami`